README.md

```diff
@@ -643,3 +643,60 @@ macro:
   elapsed_sec:
     mean: 25.911653497483996
 ```
+
+## Retrieval Evaluation Metrics
```

Comment:
I think we should put this under a section named `Tools comparison` at level 3, i.e. with 3 #.
And this should be with 4 #. Something else which I should add under `Tools comparison` is the SPARQL comparison. 

Currently this is defined with 2 # at the same level as License, Installation and Usage, which is odd

Reply:
Thank you. I will document the SPARQL comparison in a consecutive PR after we merge 9 & 10